front/app/modules/commercial/project_folders/index.tsx

```diff
@@ -185,29 +175,33 @@ const configuration: ModuleConfiguration = {
     citizen: [
       {
         path: 'folders/:slug',
-        element: <FolderShowPageComponent />,
+        element: <FolderShowPage />,
       },
     ],
-    admin: [
+    ['admin.projects']: [
       {
-        path: 'projects/folders/new',
-        element: <FolderSettingsComponent />,
+        path: 'folders/new',
+        element: <FolderSettings />,
       },
       {
-        path: 'projects/folders/:projectFolderId',
-        element: <FolderContainerComponent />,
+        path: 'folders/:projectFolderId',
+        element: <FolderContainer />,
         children: [
           {
-            index: true,
```

Comment:
The issue was here, same as what we had with the Navigation tab earlier. Just adding path: '' would have worked, but I named the route 'projects' (it's the projects inside your folder).